python/ml4ir/applications/ranking/tests/test_ml4ir_sanity.py

```diff
@@ -0,0 +1,200 @@
+import unittest
+import warnings
+import pandas as pd
+import numpy as np
+import pathlib
+from testfixtures import TempDirectory
+from ml4ir.applications.ranking.pipeline import RankingPipeline
+from ml4ir.applications.ranking.config.parse_args import get_args
+
+warnings.filterwarnings("ignore")
+
+
+def ml4ir_sanity_pipeline(df, working_dir, log_dir, n_features):
+    """
+    Train ml4ir on the passed data and read the MRR for ml4ir
+    """
+    df.to_csv(working_dir / 'train' / 'data.csv')
+    df.to_csv(working_dir / 'validation' / 'data.csv')
+    df.to_csv(working_dir / 'test' / 'data.csv')
+
+    fconfig_name = "feature_config_sanity_tests_" + str(n_features) + "_features.yaml"
+    feature_config_file = pathlib.Path(__file__).parent / "data" / "configs" / fconfig_name
+    model_config_file = pathlib.Path(__file__).parent / "data" / "configs" / "model_config_sanity_tests.yaml"
+    train_ml4ir(working_dir.as_posix(), feature_config_file.as_posix(), model_config_file.as_posix(), log_dir.as_posix())
+    ml4ir_results = pd.read_csv(log_dir / 'test_command_line'/ '_SUCCESS', header=None)
```

Comment:
[Minor] @mohazahran This should be returned by `rp.run()` as well. But this works too.